soc/nxp/imx/imx9/imx943/Kconfig.defconfig.mimx94398.m33

```diff
@@ -37,4 +37,7 @@ config IDLE_STACK_SIZE
 
 endif
 
+config DMA_TCD_QUEUE_SIZE
```

Comment:
use configdefault, refer to https://github.com/zephyrproject-rtos/zephyr/pull/98006